hack/test-integration.sh

```diff
@@ -13,14 +13,48 @@ cd "${OS_ROOT}"
 
 os::build::setup_env
 
-function cleanup()
-{
-  [ ! -z "${ETCD_STARTED-}" ] && return
-  set +e
-  kill ${ETCD_PID} 1>&2 2>/dev/null
-  rm -rf ${ETCD_DIR} 1>&2 2>/dev/null
-  echo
-  echo "Complete"
+function cleanup() {
+	set +e
+	kill ${ETCD_PID} 1>&2 2>/dev/null
```

Comment:
Also put quotes around bash arguments that include variables - `something ${FOO}` is verboten, it should always be `something "${FOO}"`